webapp/channels/src/components/advanced_text_editor/toggle_formatting_bar.test.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fireEvent, screen} from '@testing-library/react';
+import React from 'react';
+import {IntlProvider} from 'react-intl';
+
+import '@testing-library/jest-dom/extend-expect';
+import {renderWithContext} from 'tests/react_testing_utils';
```

Comment:
Nit lets use imports directly from the utils
```suggestion
import {renderWithContext, fireEvent, screen} from 'tests/react_testing_utils';
```